app/vmselect/prometheus/prometheus.go

```diff
@@ -1022,6 +1032,7 @@ type commonParams struct {
 	end              int64
 	currentTimestamp int64
 	filterss         [][]storage.TagFilter
+	limit            int
```

Comment:
How limit applied to other API requests? Why it's at common params?

Reply:
removed from common params